src/main/kotlin/net/perfectdreams/loritta/helper/serverresponses/english/ValorShipResponse.kt

```diff
@@ -0,0 +1,30 @@
+package net.perfectdreams.loritta.helper.serverresponses.english
+
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.perfectdreams.loritta.api.messages.LorittaReply
+import net.perfectdreams.loritta.helper.serverresponses.RegExResponse
+import net.perfectdreams.loritta.helper.utils.Emotes
+import java.util.regex.Pattern
+
+/**
+ * Replies to questions about ship customization
+ * on Loritta's website
+ */
+class ValorShipResponse : RegExResponse() {
+    init {
+        patterns.add("change|alter|pay|bribe|set|rise|lower".toPattern(Pattern.CASE_INSENSITIVE))
+        patterns.add("(ship)".toPattern(Pattern.CASE_INSENSITIVE))
+    }
+
+    override fun getResponse(event: GuildMessageReceivedEvent, message: String) = listOf(
+        LorittaReply(
+            "The `+ship` result value is based on the name of the two people who shipped. shippou. By changing your Discord name alterar o seu nome no Discord, the ship value will also change! The value generated is random, and persists until you change your name again.",
```

Comment:
You did not translate the whole paragraph.

Reply:
I forgot to delete the portuguese text while I was translating it (again) 

~~this is what happens when you PR at 2 in the morning before a school day~~